src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs

```diff
@@ -21,7 +21,7 @@ public PerformanceTests(ITestOutputHelper output)
         }
 
         [Theory]
-        [ActiveIssue("PerformanceTest")]
```

Comment:
We don't need [Theory], just [Benchmark], you can refer example [here](https://github.com/dotnet/corefx/blob/master/src/System.Text.Encoding/tests/Performance/Perf.Encoding.cs#L23)